### ISSUE TITLE
Install script fails on Linux

### ISSUE THREAD

**Issue by desyncr** [equalitie/ceno#54]
It has something to do with [this line](https://github.com/equalitie/ceno/blob/next/ceno-box/installCENO.sh#L35) and the current directory.

I can run CENO successfuly when I `cd` into `~/CENOBox` directory.
```
cd ~
sh installCENO.sh

<...snip...>

CENO language set to en_US.UTF-8
mktemp: failed to create directory via template ‘browser-profiles/chrome/google-chrome.XXXXXXX’: No such file or directory
rm: missing operand
Try 'rm --help' for more information.
./CENOBox/CENO.sh: line 82: cd: ceno-client: No such file or directory
./CENOBox/CENO.sh: line 83: ../CENO.log: Permission denied
Started CENOClient proxy
You are ready to use CENO.
Remember that you are covered by CENO only when the CENO Router plugin is loaded in your browser.

```

System is Fedora 22. Shell is ZSH with Tmux, if that matter.

Update:
CENOBox version v0.3.1

**Comment by misaakidis** [comment]
@desyncr thanks for reporting this, can you can replicate the issue with CENO v0.3.1? Just remove the previous installation and give this a try https://github.com/equalitie/ceno#cenobox-in-linux-and-mac-os-x  
It might indeed be an incompatibility with ZSH.

**Comment by desyncr** [comment]
Sorry, forgot to mention CENO version. It's 0.3.1. I use the following line from the README:

```
cd ~
wget -q https://raw.githubusercontent.com/equalitie/ceno/master/ceno-box/installCENO.sh && sh installCENO.sh
```

Will try with `bash` to see if there is a problem with `zsh`.

**Comment by desyncr** [comment]
[Same behavior](https://gist.github.com/desyncr/25bbc0ee183228163aad) with bash.

If I change:
```
    sh ./CENOBox/CENO.sh
```

with:
```
    cd ./CENOBox
        sh ./CENO.sh
    cd -
```

it works correctly, though.

**Comment by misaakidis** [comment]
Pushed a quick fix with this commit 9ae674acdc
Thank you for noticing this and taking the time to provide detailed logs!

**Comment by desyncr** [comment]
No problem. Works fine now.
